modules/xmpp/SdpConsistency.js

```diff
@@ -0,0 +1,139 @@
+var transform = require('sdp-transform');
```

Comment:
Why not using `import`? 

Also maybe transform should be declared with `const` or `let` if we are using ES6.